tools/cldr-apps/js/src/views/MainHeader.vue

```diff
@@ -155,9 +151,6 @@ export default {
         cldrStatus.getNewVersion() +
         " " +
         cldrStatus.getPhase();
-      if (needUpdate) {
-        this.$forceUpdate();
```

Comment:
no idea what this $forceUpdate was for -- if we don't need it, good, one less mystery to worry about